TodoApp/TodoApp.Services.Tests/Updaters/ItemUpdaterTests.cs

```diff
@@ -30,15 +30,15 @@ public void SetUp()
         {
             var dateTimeGenerator = Substitute.For<IDateTimeGenerator>();
             dateTimeGenerator.GetActualDateTime().Returns(TimeNow);
-            _itemUpdater = new ItemUpdater(dateTimeGenerator);
+            var itemRepository = Substitute.For<IItemRepository>();
+
+            _itemUpdater = new ItemUpdater(itemRepository, dateTimeGenerator);
         }
 
         [Test]
         public void UpdateItem_ItemWithTheUpdatedTimeOfLastChangeReturned()
         {
-            var mongoCollection = Substitute.For<IMongoCollection<Item>>();
-
-            var receivedItem = _itemUpdater.UpdateItemInCollection(mongoCollection, DefaultId, DefaultItem).Result;
+            var receivedItem = _itemUpdater.UpdateItem(DefaultId, DefaultItem).Result;
 
             Assert.That(receivedItem.LastChange, Is.EqualTo(TimeNow));
```

Comment:
You are testing only one property LastChange, but you should validate if whole item is as you expect it to be.